commit e572d7e6e7cf
Author: Xidorn Quan <me@upsuper.org>
Date:   Tue Aug 8 22:41:32 2017 +1000

    Bug 1388319 - Mark ServoStyleSet dirty in its Init. r=emilio
    
    MozReview-Commit-ID: 804KasIiS6E
    
    --HG--
    extra : rebase_source : 36eb21c5ffc4321aef652947455e73970419803a
---
 layout/style/ServoStyleSet.cpp | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git layout/style/ServoStyleSet.cpp layout/style/ServoStyleSet.cpp
index adb3cba922d2..a1cd15903ade 100644
--- layout/style/ServoStyleSet.cpp
+++ layout/style/ServoStyleSet.cpp
@@ -81,8 +81,9 @@ ServoStyleSet::Init(nsPresContext* aPresContext, nsBindingManager* aBindingManag
     }
   }
 
-  // No need to Servo_StyleSet_FlushStyleSheets because we just created the
-  // mRawSet, so there was nothing to flush.
+  // We added prefilled stylesheets into mRawSet, so the stylist is dirty.
+  // The Stylist should be updated later when necessary.
+  SetStylistStyleSheetsDirty();
 }
 
 void
